src/main/java/com/rackspace/ceres/app/web/MetadataController.java

```diff
@@ -95,6 +98,17 @@ public MetadataController(MetadataService metadataService, Environment environme
         .getTagValues(tenantHeader, metricName, tagKey);
   }
 
+  @GetMapping("/tags")
+  public Mono<GetTagsResponse> getTags(
+      @RequestParam(required = false) String metricName,
+      @RequestParam(required = false) String metricGroup,
+      @ApiIgnore @RequestHeader(value = "#{appProperties.tenantHeader}") String tenantHeader) {
```

Comment:
Why move this from header to tags?

Reply:
I don't understand your reply `didn't get you here` to my comment.

Reply:
added `@ApiIgnore` to tenantHeader so that it gets hidden from swagger. 

https://github.com/racker/ceres/pull/55#discussion_r656194592

Reply:
oic ok cool